src/Polly.Core/Hedging/Controller/HedgingExecutionContext.cs

```diff
@@ -94,7 +94,7 @@ public void Complete()
         // We are intentionally doing the cleanup in the background as we do not want to
         // delay the hedging.
         // The background cleanup is safe. All exceptions are handled.
-        _ = CleanupInBackgroundAsync();
+        return CleanupInBackgroundAsync();
```

Comment:
Should we remove this comment and rename the method of it's no longer done in the background?

Reply:
I have inlined `CleanupInBackgroundAsync` here, it was small anyway.